app/src/main/java/net/skyscanner/backpack/demo/compose/BpkStarRatingStory.kt

```diff
@@ -0,0 +1,132 @@
+ * limitations under the License.
+ *
+ */
+
+package net.skyscanner.backpack.demo.compose
+
+import android.content.Context
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import net.skyscanner.backpack.compose.starrating.BpkHotelRating
+import net.skyscanner.backpack.compose.starrating.BpkStarRating
+import net.skyscanner.backpack.compose.starrating.BpkStarRatingSize
+import net.skyscanner.backpack.compose.starrating.RoundingType
+import net.skyscanner.backpack.compose.text.BpkText
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.demo.R
+
+@Composable
+fun BpkStarRatingStory(
+  modifier: Modifier = Modifier,
+) {
+  val context = LocalContext.current
+  Box(
+    modifier = modifier.fillMaxHeight(),
+    contentAlignment = Alignment.Center,
+  ) {
+    Column(modifier = Modifier.padding(start = BpkSpacing.Lg)) {
+      BpkText(
+        text = stringResource(R.string.static_default_rating_title),
+        style = BpkTheme.typography.heading2,
+      )
+      Row() {
+        Column(modifier = Modifier.padding(end = BpkSpacing.Xl)) {
+          StaticRatingSample(
+            context = context,
+            size = BpkStarRatingSize.Large,
+            text = stringResource(R.string.icons_large),
+          )
+        }
+        Column() {
+          StaticRatingSample(
+            context = context,
+            size = BpkStarRatingSize.Small,
+            text = stringResource(R.string.icons_small),
+          )
+        }
+      }
+      Spacer(modifier = Modifier.padding(top = BpkSpacing.Lg))
+      Column {
+        BpkText(
+          text = stringResource(R.string.hotel_star_rating_title),
+          style = BpkTheme.typography.heading2,
+        )
+        Row {
+          Column(modifier = Modifier.padding(end = BpkSpacing.Xl)) {
+            HotelRatingSample(
+              context = context,
+              size = BpkStarRatingSize.Large,
+              text = stringResource(R.string.icons_large),
+            )
+          }
+          Column {
+            HotelRatingSample(
+              context = context,
+              size = BpkStarRatingSize.Small,
+              text = stringResource(R.string.icons_small),
+            )
+          }
+        }
+      }
+    }
+  }
+}
+
+@Composable
+private fun HotelRatingSample(context: Context, size: BpkStarRatingSize, text: String) {
+  BpkText(
+    text = text,
+    style = BpkTheme.typography.heading3,
+  )
+  var count = 1
```

Comment:
rather than having a separate `count` property you can use (or rename the `i` in the for loop as you're already looping from 1 to 5

Reply:
Haha not sure why I complicated it like that. Should be fixed now!